include/mqtt/optional.hpp

```diff
@@ -31,6 +31,11 @@ using boost::optional;
 using nullopt_t = boost::none_t;
 static const auto nullopt = boost::none;
 
+template <typename T, typename U>
```

Comment:
How does this get used?

Reply:
I forgot to erase it. At first, I thought that `boost::optional` requires in-place factory but I noticed that `boost::optional` also have `emplace()`.

I will remove it.

Reply:
Removed.